src/content/docs/en/reference/experimental-flags/raw-env-values.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Experimental raw environment variables values
+sidebar:
+  label: Raw environment variables values
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro'
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `false`<br />
+<Since v="5.12.0" />
+</p>
+
+The `experimental.rawEnvValues` flag disables coercion of `import.meta.env` values that are populated from `process.env`.
```

Comment:
```suggestion
Astro allows you to configure a [type-safe schema for your environment variables](/en/guides/environment-variables/#type-safe-environment-variables), and converts variables imported via `astro:env` into the expected type.  

However, Astro also additionally converts your environment variables imported from `import.meta.env`, and this can prevent access to some values such as the strings `"true"` (which is converted to a boolean value), and `"1"` (which is converted to a number).

The `experimental.rawEnvValues` flag disables coercion of `import.meta.env` values that are populated from `process.env`, allowing you to use the raw, imported value.
```
Once again making things up, but can we have an introduction maybe something like this?